integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt

```diff
@@ -156,33 +165,41 @@ class Sep24End2EndTest(
     wallet.stellar().submitTransaction(tx)
 
     val token = anchor.auth().authenticate(newAcc)
-    val deposits = (0..5).map { makeDeposit(token, newAcc).also { delay(7.seconds) } }
+    val deposits =
+      (0..5).map { makeDeposit(asset, amount, token, newAcc).also { delay(7.seconds) } }
     deposits.forEach { waitStatus(it, COMPLETED, token) }
-    val history = anchor.getHistory(USDC, token)
+    val history = anchor.getHistory(asset, token)
 
     Assertions.assertThat(history).allMatch { deposits.contains(it.id) }
   }
 
-  private fun `list by stellar transaction id works`() = runBlocking {
-    val token = anchor.auth().authenticate(keypair)
+  private fun `list by stellar transaction id works`(asset: StellarAssetId, amount: String) =
+    runBlocking {
+      val token = anchor.auth().authenticate(keypair)
 
-    val txId = makeDeposit(token)
+      val txId = makeDeposit(asset, amount, token)
 
-    waitStatus(txId, COMPLETED, token)
+      waitStatus(txId, COMPLETED, token)
 
-    val transaction = anchor.getTransaction(txId, token) as DepositTransaction
+      val transaction = anchor.getTransaction(txId, token) as DepositTransaction
 
-    val transactionByStellarId =
-      anchor.getTransactionBy(token, stellarTransactionId = transaction.stellarTransactionId)
+      val transactionByStellarId =
+        anchor.getTransactionBy(token, stellarTransactionId = transaction.stellarTransactionId)
 
-    assertEquals(transaction.id, transactionByStellarId.id)
-  }
+      assertEquals(transaction.id, transactionByStellarId.id)
+    }
 
   fun testAll() {
-    println("Running SEP-24 end-to-end tests...")
-    `typical deposit end-to-end flow`()
-    `typical withdraw end-to-end flow`()
-    listAllTransactionWorks()
-    `list by stellar transaction id works`()
+    println("Running SEP-24 USDC end-to-end tests...")
+    `typical deposit end-to-end flow`(USDC, "5")
+    `typical withdraw end-to-end flow`(USDC, "5")
+    listAllTransactionWorks(USDC, "5")
+    `list by stellar transaction id works`(USDC, "5")
+
+    println("Running SEP-24 USDC end-to-end tests...")
```

Comment:
This should be `XLM`.
```suggestion
    info("Running SEP-24 XLM end-to-end tests...")
```